PSP-265 - medium/small layout

coreysherman-accesso opened the pull request:
I apologize in advanced... Just remember that I am not responsible for most of this repo... This is not the cleanest code, even though I am trying to improve upon it and make it more up to our standards as we go.

In this PR: 
- added an "isFullScreenPicker" property to the directive and component
- added additional templating and styling to make the picker go full screen

This PR is for the layout only, so I left the buttons in for now to keep it fully functional, there will be a separate PR for the scrolling

coreysherman-accesso committed:
PSP-265 - medium/small layout

- removing previous attempt at flipping to a "mobile" view

coreysherman-accesso committed:
PSP-265 - medium/small layout

- implementing a new "full screen" layout for the date range picker
- cleaning up the html for the picker

coreysherman-accesso committed:
PSP-265 - medium/small layout

- implementing a new "full screen" layout for the date range picker
- cleaning up the html for the picker

coreysherman-accesso committed:
PSP-265 - medium/small layout

- implementing a new "full screen" layout for the date range picker
- cleaning up the html for the picker

jefferywroblewski-accesso commented on src/daterangepicker/components/date-range-picker.component.html:
```diff
@@ -9,88 +9,429 @@
+				</div>
+			</div>
+		</div>
+		<div class="dp-footer" *ngIf="!autoApply && (!ranges.length || (showCalInRanges && !singleDatePicker))">
```
The footer is disappearing on a small responsive layout when the height is below 740px so if we deferred the future scroll work, this still isn't a usable solution.  We would want two calendars stacked on top of each other not to push the action buttons in the footer out of the way.

I'm thinking we can mirror the same structure from PS's small layout in this component.

here's a codepen that isolates the positioning styles:
https://codepen.io/jefferywroblewski-accesso/pen/ZEGrogB

We would need to wrap the whole thing in a new container element.  Then you have to wrap the body and footer in another container to get everything in the right place with the scrollable body.
Give that a go and see if the existing contents here just fits in the flex containers.

coreysherman-accesso replied:
This is a very good catch, I applied your suggestions and it worked perfectly.  One caveat, something in the demo of this control is not playing nicely, like a width not being set somewhere, so the width of the picker gets a bit wonky, but it works perfectly in Passport Support.

coreysherman-accesso committed:
PSP-265 - medium/small layout

- implementing a new "full screen" layout for the date range picker
- cleaning up the html for the picker

coreysherman-accesso committed:
PSP-265 - medium/small layout

- implementing a new "full screen" layout for the date range picker
- cleaning up the html for the picker
